controller/schema.go

```diff
@@ -42,27 +42,32 @@ func migrateDB(db *sql.DB) error {
 )`,
 		`CREATE UNIQUE INDEX ON apps (name) WHERE deleted_at IS NULL`,
 
-		`CREATE SEQUENCE app_event_ids`,
-		`CREATE TYPE app_event_type AS ENUM ('app_deletion', 'deployment', 'job', 'scale')`,
-		`CREATE TABLE app_events (
-    event_id    bigint         PRIMARY KEY DEFAULT nextval('app_event_ids'),
-    app_id      uuid           NOT NULL REFERENCES apps (app_id),
-    object_type app_event_type NOT NULL,
+		`CREATE SEQUENCE event_ids`,
+		`CREATE TYPE event_type AS ENUM ('app_deletion', 'app', 'deployment', 'job', 'scale', 'release', 'artifact', 'provider', 'resource', 'resource_deletion', 'key', 'key_deletion', 'route', 'route_deletion')`,
+		`CREATE TABLE events (
+    event_id    bigint         PRIMARY KEY DEFAULT nextval('event_ids'),
+    app_id      uuid           REFERENCES apps (app_id),
+    object_type event_type NOT NULL,
     object_id   text           NOT NULL,
     unique_id   text,
     data        text,
     created_at  timestamptz    NOT NULL DEFAULT now()
 )`,
-		`CREATE UNIQUE INDEX ON app_events (unique_id)`,
-		`CREATE FUNCTION notify_app_event() RETURNS TRIGGER AS $$
+
+		`CREATE UNIQUE INDEX ON events (unique_id)`,
+		`CREATE FUNCTION notify_event() RETURNS TRIGGER AS $$
     BEGIN
-	PERFORM pg_notify('app_events', NEW.event_id || ':' || NEW.app_id);
+  IF NEW.app_id IS NOT NULL THEN
+    PERFORM pg_notify('events', NEW.event_id || ':' || NEW.app_id);
+  ELSE
+    PERFORM pg_notify('events', NEW.event_id || '');
```

Comment:
`event_id` is an int, the append converts it to a string which is necessary. This should probably use a more obvious method of conversion though.